console/static/surveys/lms.1.json

```diff
@@ -0,0 +1,226 @@
+{
+  "submitted_at": "2018-06-01T13:31:39.953654+00:00",
+  "survey_id": "lms",
+  "case_ref": "1000000000000001",
+  "data": [
+    {
+      "answer_instance": 0,
+      "value": "Yes",
+      "answer_id": "address-check-answer",
+      "block_id": "address-check-block",
+      "group_id": "about-household-group",
+      "group_instance": 0
+    },
+    {
+      "answer_instance": 0,
+      "value": "Jimmy",
+      "answer_id": "first-name",
+      "block_id": null,
+      "group_id": null,
```

Comment:
@nealedj is going to look into why these values are null

Reply:
David has raised a card on the eQ side to investigate this. For now, this is the data that will be generated by eQ.